src/rest/__tests__/fixtures/dataFromPreviewApp.json

```diff
@@ -18,7 +18,7 @@
     "shortTextUrl": "https://example.com",
     "shortTextSlug": "all-fields-page",
     "shortTextList": ["short", "text", "list", "comma"],
-    "longText": "long\ntext\nmultiple",
+    "longTextMultiple": "long\ntext\nmultiple",
```

Comment:
Was also not working in the test, as there was no match with the content type yet